app/controllers/alchemy/json_api/pages_controller.rb

```diff
@@ -117,6 +121,17 @@ def api_page(page)
         Alchemy::JsonApi::Page.new(page, page_version_type: page_version_type)
       end
 
+      def etag(pages)
+        pages = Array.wrap(pages)
+        return unless pages.any?
+        relevant_params = params.to_unsafe_hash.slice(*JSONAPI_STALEMAKERS).values.compact
```

Comment:
Very theoretically, but could it be that two keys have the same value?
Shouldn't we include the keys in the cache key?

```
?include=author;sort=author
```

_hard to make up a case honestly, but still possible, no?_